### DISCUSSION TITLE
Navbar mobile menu dropdown not working. [React]

### DISCUSSION THREAD

**Discussion by soumik9** [themesberg/flowbite-react#562] [General]
I have install tailwind and flowbite,

in tailwind.config.js,

module.exports = {
  content: [
    "./src/**/*.{html,js}"
  ],
  theme: {
    extend: {},
  },
  plugins: [
    require('flowbite/plugin')
  ],
}

and imported flowbite in index.js. Is there anything wrong? When i try for mobile the navbar toggle button not working.

**Comment by zoltanszogyenyi** [comment]
For interactive components please check out the Flowbite React library:

https://github.com/themesberg/flowbite-react

**Comment by marcelo-earth** [comment]
I'm having the same problem using the Flowbite React library.

**Comment by tulup-conner** [comment]
>and imported flowbite in index.js.

If you're doing this, that's where you are going wrong. You want to import components from `flowbite-react`. For example, you can create a `<Navbar>` in React using the code examples at https://flowbite-react.com/navbars.

>I'm having the same problem using the Flowbite React library.

Can you provide more context?

BTW, the steps to get this setup are at https://github.com/themesberg/flowbite-react#getting-started.

**Comment by paustria** [comment]
Same here. I tried on next.js with react 18. The toggle is not working.
![navbar](https://user-images.githubusercontent.com/1796066/219850694-60e83853-ddbc-47ff-aa96-89ed7cc34eca.gif)

**Reply by paustria** [reply]
Oh boy.. I found out the reason why the toggle didn't work in my case. I was putting the `<Navbar />` component in `_document.tsx`. 😭

**Reply by marcelo-earth** [reply]
😅 Yes, is that. I was about to suggest that when I read this comment.

**Reply by Taher2512** [reply]
Can you explain it please? I'm new to tailwind and can't understand what you mean by "<Navbar /> component in _document.tsx" 😅
